### PR TITLE
Add All Full App Features 

### PR DESCRIPTION
**Pull request by yayner2002**
In this PR, The following Tasks were done, Hope you will 👍  it.

- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions, and reducers in React.
- Add Video Description Of the project [follow this link for video description](https://loom.com/share/83b1fc8004b24551912ca63ea8ca0506)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,6 @@ This web App uses a real live data from the Blockchain API. It's a web applicati
 - Redux Toolkit
 - Bootstrap
 - CSS
-
-## Video description About the project
-[follow this link for video description](https://loom.com/share/83b1fc8004b24551912ca63ea8ca0506)
 ## Screenshot
```
Comment (rustam-tolipov) [inline README.md:12]: - [x] Please add Acknowledgements to your readme

File: README.md [hunk 2]
```diff
@@ -8,9 +8,6 @@ This web App uses a real live data from the Blockchain API. It's a web applicati
...
```
Comment (rustam-tolipov) [inline README.md:11]: - [x] As a requirement items could be filtered by given parameter or category name. You can add a button to sort them by price or names in order.

_In requirements_
one page with a list of items that could be filtered by some parameters; in the example below, it's a list of metrics that can be filtered by the country (imagine a search field to introduce the country name like Italy, Croatia, etc.). This page should be your homepage.

File: README.md [hunk 3]
```diff
@@ -8,9 +8,6 @@ This web App uses a real live data from the Blockchain API. It's a web applicati
...
 
 ![mockup](/src/assets/images/cryptomobile.png?raw=true "mockup")
```
Comment (rustam-tolipov) [inline README.md:13]: - [ ] You have one failing test. Kindly make sure your all tests are passing.


Screen Shot 2022-09-02 at 04 18 25

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @yayner2002, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @yayner2002  👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
alt

 ## Highlights
- All tests are passing 👍 
- Nice use of git-flow ✔️ 
- Good commit messages ✔️ 


 ## Required Changes ♻️

- [ ] Kindly as requested by the previous reviewer, try to make your project more user-engaging by adding a search feature, where you can filter the results by some parameters like category, etc .you got this 👍 

_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hello @yayner2002 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**